src/main/java/com/gmail/goosius/siegewar/timeractions/DefenderWin.java

```diff
@@ -22,7 +24,13 @@
 	 * @param winnerTown the winning town
 	 */
     public static void defenderWin(Siege siege, Town winnerTown) {
-        SiegeWarSiegeCompletionUtil.updateSiegeValuesToComplete(siege, SiegeStatus.DEFENDER_WIN);
+		SiegeWarSiegeCompletionUtil.updateSiegeValuesToComplete(siege, SiegeStatus.DEFENDER_WIN);
+		
+		if (winnerTown.hasNation())
+			try {
+				NationMetaDataController.addWinOrLoss(winnerTown.getNation(), true);
+			} catch (NotRegisteredException ignored) {}
+		NationMetaDataController.addWinOrLoss(siege.getAttackingNation(), false);
 
 		Messaging.sendGlobalMessage(Translation.of("msg_siege_war_defender_win", winnerTown.getFormattedName()));
 
```

Comment:
:construction: 
- Above in this PR, there are 2 scenarios in this PR for win/loss stats change, but these 2 scenarios don't include early win scenarios.
- Early win scenarios should also be counted i.e.
   - SurrenderTown  (here the attacker can be considered the winner)
   - AbandonAttack  (here the defender can be considered the winner)